pkg/utils/util.go

```diff
@@ -338,3 +339,19 @@ func ApiextensionsJsonToKyvernoConditions(original apiextensions.JSON) (interfac
 	}
 	return nil, fmt.Errorf("error occurred while parsing %s: %+v", path, err)
 }
+
+func OverrideRuntimeErrorHandler() {
+	logger := log.Log.WithName("RuntimeErrorHandler")
+	if len(runtime.ErrorHandlers) > 0 {
+		runtime.ErrorHandlers[0] = func(err error) {
+			logger.V(6).Info("Runtime error occurred", err)
```

Comment:
We can remove `occurred` from the message and change to:

```
			logger.V(6).Infof("runtime error: %s", err)
```

Reply:
fixed